infrastructure/environments/monitoring.tf

```diff
@@ -105,22 +105,25 @@ XML
 }
 
 resource "azurerm_monitor_action_group" "monitoring" {
-  count = local.monitoring_alert_email_count
+  count = local.monitoring_alert_email_enabled_count
 
   name = format(local.name_format, "monitoring_critical_alert")
   resource_group_name = azurerm_resource_group.monitoring.name
   short_name = substr("PINS ${local.workspace_name}", 0, 12)
   enabled = true
 
-  email_receiver {
-    email_address = var.monitoring_alert_email
-    name = "PINS DevOps"
-    use_common_alert_schema = true
+  dynamic "email_receiver" {
+    for_each = var.monitoring_alert_emails
+    content {
+      email_address = email_receiver.value
+      name = "Alerts for ${email_receiver.value}"
+      use_common_alert_schema = true
+    }
   }
 }
 
 resource "azurerm_monitor_metric_alert" "web_ping_test" {
-  count = local.monitoring_ping_tests_count
+  count = local.monitoring_alert_email_enabled_count == 1 && local.monitoring_alert_email_enabled_count == 1 ? 1 : 0
```

Comment:
`local.monitoring_alert_email_enabled_count == 1 && local.monitoring_alert_email_enabled_count == 1`

same thing ?

Reply:
Believe it or not, I noticed this at the same time. Well spotted